doc/api/events.markdown

```diff
@@ -104,7 +104,7 @@ Note that `emitter.setMaxListeners(n)` still has precedence over
 
 ### emitter.listeners(event)
 
-Returns an array of listeners for the specified event.
+Returns a copy of an array of listeners for the specified event.
```

Comment:
That would be lying though. Sometimes that array doesn't exist until you call .listeners(event)